Where_Is_My_Wife/Assets/_Main/Scripts/Dialogue/DialogueManager.cs

```diff
@@ -0,0 +1,70 @@
+using TMPro;
+using UnityEngine;
+using WhereIsMyWife.Dialogue;
+
+namespace WhereIsMyWife.Managers
+{
+   public class DialogueManager : Singleton<DialogueManager>
```

Comment:
I think this system is far to complex to what we may be using, localization will not be necessary for this project, not in scope, remove it, we can figure an easier solution in the near future.

Reply:
96665c08c08aa3776ec96d7030a7f19c1066d17c